### ISSUE TITLE
Styled MDX tables do not respect the specified alignment.

### ISSUE THREAD

**Issue by Everspace** [system-ui/theme-ui#654]
**Describe the bug**
The styled MDX is unaligned, regardless of how you specify it.

**To Reproduce**
1. Use a gatsby theme-ui/mdx starter
2. Use the following MDX

```mdx
| Left | Centre | Right |
|:-----|:------:|------:|
| 1 | 2 | 3 |
```

3. Navigate to page with generated MDX

**Expected behavior**
The `th`,  and `td` elements should have the `align` attribute correctly set.

In addition it seems the class attribute is set strangely `class="css-0"`.

**Additional context**

Initially reported in the MDX here: https://github.com/mdx-js/mdx/issues/930

**Comment by jxnblk** [comment]
Thanks! This sounds like a bug. If anyone wants to open a PR with a failing test case, please feel free to do so

**Comment by Sporandum** [comment]
Hello,
I saw the same issue. Personally I solve the problem by customize MDX tags <th /> and <td /> like this :

```js
// /src/gatsby-plugin-theme-ui/components.js

/** @jsx jsx */
import { jsx, Styled } from "theme-ui"
import Prism from "@theme-ui/prism"

export default {
  th: ({ align, ...props }) => (
    <Styled.th style={align && { textAlign: align }} {...props} />
  ),
  td: ({ align, ...props }) => (
    <Styled.td style={align && { textAlign: align }} {...props} />
  ),
}
```

**Comment by atanasster** [comment]
Fixed with #1347
